components/composable/notebook.js

```diff
@@ -0,0 +1,399 @@
+// @flow
+import React from "react";
+
+import SyntaxHighlighter from "react-syntax-highlighter";
+import ReactMarkdown from "react-markdown";
+import { idea } from "react-syntax-highlighter/dist/styles";
+
+export const Passthrough = (props: { children?: React$Node }) =>
+  props.children || null;
```

Comment:
I might be missing something but why is this needed?

Reply:
This was here for prototyping so that I could make new elements by just writing:

```js
const CellOptions = Passthrough
const CellCreatorToolbar = Passthrough
```

Which let me think on the interface inside of the compound components without it actually affecting anything yet:

```js
<Notebook>
  <CellCreatorToolbar />
  <Cell>
    <CellOptions />
    <Input>
      <Prompt />
      <Editor />
    </Input>
    <Outputs />
  </Cell>
  <CellCreatorToolbar />
</Notebook>
```